x/ibc/02-client/client/cli/query.go

```diff
@@ -0,0 +1,241 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	tmtypes "github.com/tendermint/tendermint/types"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/types"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/types/tendermint"
+	ics23 "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment"
+	"github.com/cosmos/cosmos-sdk/x/ibc/23-commitment/merkle"
+)
+
+// GetQueryCmd returns the query commands for IBC clients
+func GetQueryCmd(queryRouter string, cdc *codec.Codec) *cobra.Command {
+	ics02ClientQueryCmd := &cobra.Command{
+		Use:                        "client",
+		Short:                      "IBC client query subcommands",
+		DisableFlagParsing:         true,
```

Comment:
Why are we `DisableFlagParsing` here?

Reply:
idk... this is legacy code

Reply:
I looked into the other modules `cli` and this is also the case. Let's open a separate issue to tackle this